src/solvers/dgmulti.jl

```diff
@@ -1,7 +1,7 @@
 # includes solver files for DGMulti solvers
 include("dgmulti/types.jl")
 include("dgmulti/dg.jl")
-include("dgmulti/flux_differencing_gsbp.jl")
+include("dgmulti/flux_differencing_GaussSBP.jl")
```

Comment:
```suggestion
include("dgmulti/flux_differencing_gausssbp.jl")
```
I don't think we should start using capitalized file names.